### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Device login_Screen Reader: Narrator/NVDA is not reading the position of the ‘Copy & Open’ and ‘Cancel’ buttons as arrow keys are used for navigating

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3318]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Open Java EE - Eclipse application and start Narrator/NVDA
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Verify if Narrator/NVDA is reading the position of the buttons as arrow keys are used to navigate to the buttons 

**Actual Result:** 
Narrator/NVDA is not reading the position of the ‘Copy & Open’ and ‘Cancel’ buttons as arrow keys are used for navigating

**Expected Result:** 
Narrator/NVDA should read the position of the ‘Copy & Open of 1 of 2’ and ‘Cancel of 2 of 2’ buttons as arrow keys are used to navigating to the buttons.

**User Impact:** 
Screen Reader users will not know that there is another button present beside ‘Cancel’ button if screen reader does not read the position of the buttons

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_5_screen reader is not reading the position of buttons.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451147/Bug_5_screen.reader.is.not.reading.the.position.of.buttons.pptx)

**Comment by Flanker32** [comment]
This is a common behavior of NVDA, it will not report the position or numbers for button. Here is a gif using NVDA read NVDA settings, as we can see, it reads the button text only.

![3318](https://user-images.githubusercontent.com/12445236/64104736-65a42700-cda7-11e9-9c3b-f1ad8971c317.gif)

**Comment by Flanker32** [comment]
- Importance of experience: Low, NVDA will tell user they meet a button and users could know there may be more buttons here.
- Quality of experience: not impacted: It's a common behavior of NVDA and users could know there may be more buttons when hearing button.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Tanushagottipati** [comment]
Regressed in the below build and the now user can able to navigate with the help of tab key. as arrow keys are no longer used to navigate there is no need for screen reader to read the position. Hence closing the bug and attached video for reference
Environment Details:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0.201909020904
[Bug Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3578524/Bug.Fixed.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749742
